apps/prairielearn/src/pages/studentInstanceQuestion/studentInstanceQuestion.ts

```diff
@@ -323,7 +323,7 @@ router.get(
     await getAndRenderVariant(variant_id, null, res.locals as any);
 
     await logPageView('studentInstanceQuestion', req, res);
-    await setQuestionCopyTargets(res);
+    const questionCopyTargets = await getQuestionCopyTargets(res);
```

Comment:
I'm trying to think through the implications of showing this on the student instance question page. This means that we'd show this link even to students when they're taking an exam if they're accessing a question that's publicly shared?

I guess this isn't a new behavior, I'm just not sure if it's good. I guess the argument to _show_ it is that if a course is a template course, instructors might want to be able to copy questions from their instance?

I hope we'll get rid of the notion of template courses eventually in favor of content sharing, so maybe this is just a weird intermediate state. But if it is, we might want to consider limiting this to only be shown for template courses, not for random questions that happen to be publicly shared.

Reply:
This line here should be ensuring that we are not showing the copy button to students, in particular the part that makes sure the question context is `'public'`: 

https://github.com/PrairieLearn/PrairieLearn/blob/b45c62ba38332d1f6e452bfc999b231ecee5be5d/apps/prairielearn/src/components/QuestionContainer.html.ts#L679-L683

Reply:
Ah! Yes, I think that does it.

So once we get rid of template courses, we can drop the call to `setQuestionCopyTargets(...)` from all non-public-preview pages, correct?